be/src/storage/rowset/bitmap_index_writer.cpp

```diff
@@ -173,6 +218,13 @@ class BitmapIndexWriterImpl : public BitmapIndexWriter {
     uint64_t size() const override {
         uint64_t size = 0;
         size += _null_bitmap.getSizeInBytes(false);
+        for (RoaringWrapper* roaring_wrapper : _size_changed_roaring_vec) {
+            uint64_t new_size = roaring_wrapper->roaring_size();
```

Comment:
You make the caller to call different function to get right size, this is not a good way.
You should make it in the RoaringWrapper do it itself.